src/Analysis/Engine/Test/Microsoft.Python.Analysis.Engine.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project>
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
-    <RootNamespace>Microsoft.PythonTools.Analysis</RootNamespace>
+    <RootNamespace>AnalysisTests</RootNamespace>
```

Comment:
Let's not change namespaces for now.

Reply:
This only affects what namespace VS uses when you create a new file. It was set to AnalysisTests in PTVS but, I guess, was overlooked when porting, which in turn accidentally caused my new file InheritanceTests to get wrong namespace.